.vscode/settings.json

```diff
@@ -4,7 +4,6 @@
     "**/dist": true
   },
   "files.exclude": {
-    "**/dist": true,
```

Comment:
@valerybugakov out of interest what is the purpose of this change?

Reply:
I should have tagged you or other folks on that. It allows me to inspect built artifacts using the file tree sidebar. We bumped into some issues related to how providers are bundled and then dynamically loaded in the VS Code extension. 

I can revert if it messes up your local setup.

Reply:
Gotcha!